schemas/publicKey.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is good, we'll probably want a JSON Schema that puts all of this together.

Reply:
I can commit to providing that.

Reply:
As a general rule, as an Editor... never resolve a conversation if you didn't open the conversation. Your intentions could have been perfectly fine, but the person commenting may misread it as you dismissing one of their concerns before they had a chance to raise it. In this case, it was fine, but you'll find that in most cases, dealing with other people, it's not fine and leads to more work for you (because now you're dealing with an issuer submitter that's pissed at you for batting their comment aside).